frontend/webpack.config.js

```diff
@@ -180,21 +180,33 @@ module.exports = () => ({
     new Html({
       template: path.resolve(__dirname, 'index.ejs'),
       inject: false,
+      templateParameters: {
```

Comment:
Sorry for mixing you up. This way is absolutely useless for us and we can remove these params passing to the template.

Reply:
This option allows use `REMARK_URL` instead of `htmlWebpackPlugin.options.REMARK_URL`

https://github.com/jantimon/html-webpack-plugin#:~:text=templateParameters

Reply:
Yep, I understand. But it's useless because we should to replace strings only on running time with passed vars to env and this template compilation will be done on building stage.